<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoRecord</class>
 <widget class="QWidget" name="VideoRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>流速--录制</string>
  </property>
  <property name="windowIcon">
   <iconset resource="flowrate.qrc">
    <normaloff>:/icons/image/icon.ico</normaloff>:/icons/image/icon.ico</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="40,10,15">
   <item>
    <widget class="QOpenGLWidget" name="frame_displayer">
     <property name="minimumSize">
      <size>
       <width>328</width>
       <height>246</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1280</width>
       <height>960</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0,0,0,0,1,0,0,0,0,0,0,1,0,1">
     <item row="1" column="1">
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:red;
border-radius:6px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBoxAutoRecord">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>智能录制</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>23</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>这是什么？</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
background-color:rgb(130, 201, 255);
}
QPushButton:pressed
{
border:1px solid black;
background-color:rgba(130, 201, 255, 128);
}</string>
          </property>
          <property name="text">
           <string>？</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="continueRecordNumLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>连续录制视频数量</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QComboBox" name="continueRecordNum">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="residueRecordCount">
       <property name="text">
        <string>剩余录制数量：0</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>亮度：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="sharpnessLabel">
       <property name="text">
        <string>对焦：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>稳定性：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="sharpnessQuality">
       <property name="minimumSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:red;
border-radius:6px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="sharpnessDurationLabel">
       <property name="text">
        <string>持续时间：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="sharpnessDurationQuality">
       <property name="minimumSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:red;
border-radius:6px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_12">
       <property name="minimumSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:red;
border-radius:6px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="totalQuality">
       <property name="minimumSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:red;
border-radius:6px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_11">
       <property name="minimumSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:red;
border-radius:6px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="totalQualityLabel">
       <property name="text">
        <string>图像综合质量：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>内容：</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QComboBox" name="flowrateProcessNumber">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentText">
        <string>2</string>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="14" column="0">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="flowrateProcessNumberLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>处理线程数</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="16" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>38</width>
         <height>139</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="12" column="0" colspan="2">
      <widget class="QStackedWidget" name="begin_stop_record_stack">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_begin_record_btn">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QPushButton" name="begin_record_btn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>录制</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_stop_record_btn">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QPushButton" name="stop_record_btn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="13" column="0" colspan="2">
      <widget class="QLabel" name="video_time_display">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_4" columnstretch="2,1,1">
     <item row="0" column="0">
      <widget class="QLineEdit" name="importDir">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>请输入导入文件夹路径</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="importButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>导入</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="recursivePathCheck">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>路径递归</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QPushButton" name="exportAllData">
       <property name="text">
        <string>导出所有详细数据</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QTableWidget" name="videorecord">
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>80</number>
       </attribute>
       <column>
        <property name="text">
         <string>视频路径</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>时长(s)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>帧率(fps)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>像素尺寸(um/px)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>放大倍率</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>计算状态</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>回车进入详细数据分析...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="flowrate.qrc"/>
 </resources>
 <connections/>
</ui>
